plugins/modules/vmware_dvswitch.py

```diff
@@ -150,6 +146,25 @@
             vlan_mtu_interval: 0,
             teaming_failover_interval: 0,
         }
+    network_policy:
+        version_added: '1.11.0'
+        description:
+            - Dictionary which configures the different default security values for portgroups.
+        suboptions:
```

Comment:
As a note, could you please add the following point?  
I confirmed that the network_policy is inherited(overwritten) to the existing all distributed port group in a dvSwitch if we update the network_policy of the dvSwitch.